js/app.js

```diff
@@ -0,0 +1,80 @@
+// 1. Verbal questions
+
+// a parameter is a variable that lives inside of a method
+// an argument is the value of after its been passed through the method
+// is it possible then to have your parameter and agrument be the same?
+// thats what it looks like in my Palindrome?
+
+// return is giving you the value of the function 
+// where console.log is just showing the output in the console
+
+// Other than returing a function value you can reuse it, pass it through an argument?
+
+
+// 2. Palindrome
+const checkPalindrome = (word) => {
+	console.log(word.split("").reverse().join(""))
+	word = word.toLowerCase()
+	const reversedWord = word.toLowerCase().split("").reverse().join("")
+	return reversedWord === word;
+	}
```

Comment:
check your indentation.  check out this site: https://beautifier.io/
paste your answer to each problem, press the button, and make your code's indentation and curly brace placement match the output on that site.  do this anywhere i mark "indentation"